server/resp/src/main/java/org/infinispan/server/resp/commands/connection/MemoryEntrySizeUtils.java

```diff
@@ -17,6 +18,11 @@ public static long calculateSize(byte[] key, InternalCacheEntry<byte[], Object>
 
    static long internalCalculateSize(byte[] key, Object value) {
       try {
+         if (value instanceof JsonBucket jb) {
+            // JsonBucket is a special case, we need to use the sizeOf library
+            // to calculate the size of the JsonBucket
+            return sizeof.deepSizeOf(key) + sizeof.deepSizeOf(jb.value());
```

Comment:
Should we had a bit of overhead for the JsonBucket class to compensate?

Reply:
yeah, I need to check how records are represented in memory